### PR TITLE
Feature/psa initial updated data model

### PR DESCRIPTION
**Pull request by katherinebernstein**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -55,6 +55,7 @@
     "babel-preset-react": "^6.24.1",
     "babel-preset-stage-2": "^6.24.1",
     "better-npm-run": "0.0.15",
+    "css-loader": "^0.28.5",
```
Comment (UnsungHero97) [inline package.json]: Please undo the changes to `webpack.config.base.js`, remove `postcss.config.js`, and remove all of these:

`css-loader`
`extract-text-webpack-plugin`
`postcss-import`
`postcss-loader`
`precss`
`style-loader`

`styled-components` lets us avoid all of this complexity around CSS. You should be able to achieve the same effect as `rowSelectionModal` using `styled-components`.

Comment (katherinebernstein) [reply package.json]: how do you pass in a class name string using styled-components?
